assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -187,11 +272,12 @@ export default function UserInputQuestionnaire() {
 				</UserInputQuestionWrapper>
 			) }
 
-			{ activeSlug === 'preview' && (
+			{ activeSlug === 'preview' && ! isSavingSettings && ! isNavigating && (
```

Comment:
This is not needed because if either `isSavingSettings` or `isNavigating` is true, we will return earlier.

```suggestion
			{ activeSlug === 'preview' && (
```